src/main/java/com/codeborne/selenide/ConditionWaiter.java

```diff
@@ -0,0 +1,56 @@
+package com.codeborne.selenide;
+
+import com.codeborne.selenide.ex.ConditionNotMatchException;
+import com.codeborne.selenide.ex.UIAssertionError;
+import org.awaitility.core.ConditionTimeoutException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.function.Predicate;
+
+import static org.awaitility.Awaitility.await;
+
+/**
+ * Class provide conditional polling wait by predicate with ignore exceptions.
+ */
+
```

Comment:
is it ok that there's an empty line between JavaDoc and the class's declaration?

Reply:
removed blank line